README.md

```diff
@@ -105,6 +105,7 @@ If you would like to contribute to the project, see [Contributing.md](.github/Co
 - Rudder keybindings not working (you have to set your keybinding to rudder axis right and left)
 - Wing dips on landing (due to bad transition to direct law in flare, same with the default A320)  [Workaround use minimal aileron input on landing]
 - Black screens/unable to start (conflict with another mod/livery or incorrect installation of the A32NX mod, use the [installer](https://api.flybywiresim.com/installer))
+- Invisble plane, no sounds? Reinstall A32NX, delete any old version from your Community folder.

```

Comment:
Typo: Invisible. Note that `Community` is surrounded by backticks in the rest of this file as far as I can see :).

Reply:
Will fix soon 😁